Makefile

```diff
@@ -153,29 +158,73 @@ else
 KIND=$(shell which kind)
 endif
 
+# Prints relevant environment variables
+.PHONY: envs
+envs:
+	@{ \
+	echo "CONTROLLER_GEN=$(CONTROLLER_GEN)"; \
+	echo "KUSTOMIZE=$(KUSTOMIZE)"; \
+	echo "AUTHORINO_IMAGE=$(AUTHORINO_IMAGE)"; \
+	echo "AUTHORINO_NAMESPACE=$(AUTHORINO_NAMESPACE)"; \
+	echo "AUTHORINO_DEPLOYMENT=$(AUTHORINO_DEPLOYMENT)"; \
+	}
+
+# Creates a namespace where to deploy Authorino
+.PHONY: namespace
+namespace:
+	kubectl create namespace $(AUTHORINO_NAMESPACE)
```

Comment:
This was originally handled by the `deploy` Makefile task as the `config` directory included the autogeneration of the namespace

Reply:
Not really. Although the `config` included a `Namespace` resource, due to the use of kustomize – particularly kustomize's `namespace` and `namePrefix` directives –, the `Namespace` resource included in `config` was completely useless. `deploy` ended up creating a namespace `authorino-system` and adding metadata of all resources `namespace: authorino`. The tools we added afterwards to deploy locally had to create the `authorino` namespace and `authorino-system` was never used.

Reply:
Wouldn't changing the `namespace` value in `config/default/kustomization.yaml` would have fixed the issue you comment

Reply:
I guess if we set `namespace: authorino-system` in `config/default/kustomization.yaml`, that would do the trick. However, having a new namespace created to deploy Authorino not always makes sense.

For cluster-wide deployment, I'd say having an "authorino" (or "authorino-system") namespace is desirable. For namespaced deployment, on the other hand, that makes less sense, as the namespace will usually be something defined by the user, eventually created beforehand with the APIs to be protected already running in there.

Anyway, this dichotomy made me want to remove the `Namespace` resource and leave it for the user to decide.

Reply:
To cover the use case of having a custom namespace name I think the `kustomize edit set` subcommand might fill that purpose.